cabal.haskell-ci

```diff
@@ -1,4 +1,5 @@
 branches: master
+installed: +all
```

Comment:
This seems to anyway be the default (I hadn't been sure about this), so this line changes nothing.

Reply:
Good to know.